README.md

```diff
@@ -136,41 +136,39 @@ To deploy your local workbench API code to a given AppEngine project, in the api
 directory run:
 
 ```
-./project.rb deploy-api --project PROJECT --account ACCOUNT@pmi-ops.org --version VERSION [--promote]
+./project.rb deploy --project PROJECT --version VERSION --[no-]promote
 ```
 
+Note: This deploys both the API and public API. Use `deploy-api` or
```

Comment:
We may want to have a place in the README explicitly where we cover how to e.g. deploy api, public-api, and ui to stable.

You may also want to mention this updates the configuration and database schema.

Reply:
Added a comment about the schema. Will add the manual stable deploy in a follow-up. I want to put some more logic in place around checking out the tagged version, switching branches, and hopefully guarding against local code changes.

Reply:
Note we now do the local change check for RDR in a few places during deployment:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/tools/deploy_app.sh#L38